{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/auth/api.py

```diff
@@ -7,6 +7,7 @@
 from {{cookiecutter.main_module}}.base import response
 from {{cookiecutter.main_module}}.base.api.mixins import MultipleSerializerMixin
 from {{cookiecutter.main_module}}.users.services import create_user_account, get_and_authenticate_user
+from {{cookiecutter.main_module}}.users.auth.services import send_password_reset_mail
```

Comment:
Let's use relative import here?

Reply:
namespaced relative service import to auth_services